ui/react-ui/src/Pages/Users.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react'
+
+class Users extends Component{
+
+    constructor(props){
+      super(props)
+      this.state={
+        users:[],
+        isLoading:false,
+        isError: false
+      }
+    }
+    // Async function to get request 
+    async componentDidMount(){
+      this.setState({isLoading:true})
+      const response = await fetch("https://nate-temp-bucket.s3-us-west-2.amazonaws.com/nate_sample.json")
```

Comment:
Maybe the fetch url should be turned into a const so when a real one is made it is easier to know what to change.